gemini/js-web-genkit/idx-template.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Genkit",
+  "description": "A sample app for showing using Gemini with Genkit",
+  "categories": ["Web app", "Backend"],
```

Comment:
Also needs to include the AI ML tag like the other Gemini template

Reply:
good call, updated